packages/api/package.json

```diff
@@ -35,6 +35,7 @@
   "devDependencies": {
     "@oakai/eslint-config": "*",
     "@oakai/prettier-config": "*",
+    "@types/archiver": "^6.0.3",
```

Comment:
This was the missing type package that revealed the download all bug

Reply:
@JBR90 Potentially relevant to your PR